audit.json

```diff
@@ -77,5 +77,6 @@
  "10096_Timestamp Disclosure - Unix_http://em-npa-aat.service.core-compute-aat.internal/api/markups/documentId/redactionId_DELETE":"ignore",
  "10096_Timestamp Disclosure - Unix_http://em-npa-aat.service.core-compute-aat.internal/api/markups/documentId_DELETE":"ignore",
  "10096_Timestamp Disclosure - Unix_http://em-npa-aat.service.core-compute-aat.internal/api/redaction_POST":"ignore",
- "10096_Timestamp Disclosure - Unix_http://em-npa-aat.service.core-compute-aat.internal/api/markups/documentId?offset=10\u0026pageNumber=10\u0026pageSize=10\u0026paged=true\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET":"ignore"
+ "10096_Timestamp Disclosure - Unix_http://em-npa-aat.service.core-compute-aat.internal/api/markups/documentId?offset=10\u0026pageNumber=10\u0026pageSize=10\u0026paged=true\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET":"ignore",
+ "100000_A Client Error response code was returned by the server_http://em-npa-aat.service.core-compute-aat.internal/v2/api-docs_GET":"ignore"
```

Comment:
with spring boot upgrade it is now using sprinf docs. and it is using `/v3/api-docs`.if you can change existing opens to v3 it should fix.